packages/shield-controller/src/backend.test.ts

```diff
@@ -1,5 +1,8 @@
+/* eslint-disable jest/no-conditional-in-test */
```

Comment:
If we add lint ignores, we should always also add a comment, why these are necessary.

In this case, it looks like this has been added because the `fetchMock` implementations use `if then else`. Would it be possible to rewrite these mocks without using conditionals?